docs/main/kafka-knative-broker/README.md

```diff
@@ -0,0 +1,102 @@
+# Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+
+Kafka perfectly fullfills this reliability need.
+
+## Pre-requisites
+
+1. A Kafka cluster running, see https://strimzi.io/quickstarts/ for a quickstart setup
+
+## Installation steps
+
+1. Configure and enable kafka broker feature in Knative: https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/
+```console
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-controller.yaml
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-broker.yaml
+```
+> [!NOTE]
+> This examples uses`knative-v1.14.5`, this may change, please refer to [the official documentation link](https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/)
```

Comment:
is it still knative-v1.14.5? the following link points to 1.16

Reply:
I rephrased the sentence to state which version was used when writing the doc  and to refer to latest doc for more up-to-date setup